docs-site/docs/user/organization/members/group-and-project-members.md

```diff
@@ -85,9 +85,10 @@ Prerequisite:
 
```

Comment:
Above this can you fix the wording of `Edit member for a group or project` to `Edit members for a group or project`

Reply:
Fixed in 52b5939cbce0f838bffc5b200d96f444c0d5756d